srcalc/src/test/java/gov/va/med/srcalc/web/view/VariableEntryTest.java

```diff
@@ -75,28 +102,52 @@ public final void testRetrievedCardiacAge()
     @Test
     public final void testWithRetrievedLabs()
     {
-        final List<AbstractVariable> vars = SampleModels.sampleVariableList();
+        final List<AbstractVariable> vars = new ArrayList<AbstractVariable>();
         final Patient patient = SampleCalculations.dummyPatientWithLabs(1);
-        final VariableEntry entry = VariableEntry.withRetrievedValues(vars, patient);
-        final HashMap<String, String> expected = new HashMap<>();
-        
-        expected.put(vars.get(1).getKey(), String.valueOf(patient.getAge()));
-        expected.put(vars.get(2).getKey(), patient.getGender());
-        expected.put(vars.get(5).getKey(), VariableEntry.SPECIAL_NUMERICAL);
         
-        final RetrievedValue labValue = patient.getLabs().get("WBC");
-        
-        final String retrievalString = VariableEntry.makeRetrievalString(
-                labValue.getValue(),
-                labValue.getMeasureDate(),
-                labValue.getUnits());
+        final HashMap<String, String> expected = new HashMap<>();
         
-        expected.put(VariableEntry.makeNumericalInputName(vars.get(5).getKey()),
-                String.valueOf(labValue.getValue()));
-        expected.put(
-                VariableEntry.makeRetrievalString(
-                        VariableEntry.makeNumericalInputName(vars.get(5).getKey())),
-                retrievalString);
+        addAllLabs(expected, vars, patient);
+        final VariableEntry entry = VariableEntry.withRetrievedValues(vars, patient);
         assertEquals(expected, entry.getDynamicValues());
     }
+    
+    private void addAllLabs(final Map<String, String> expected, final List<AbstractVariable> vars,
```

Comment:
This method is complex enough to warrant a Javadoc. I had to read it through to understand what it was doing.

It's pretty specific to `testWithRetrievedLabs()`, does it really deserve to be a separate method at all?